<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск кратчайшего пути</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3" rowspan="4">
     <widget class="QGraphicsView" name="graphView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxVertices">
      <property name="title">
       <string>Вершины</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutVertices">
       <item>
        <widget class="QLineEdit" name="verticesCountLineEdit">
         <property name="placeholderText">
          <string>Введите количество вершин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setVerticesCountButton">
         <property name="text">
          <string>Установить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxPath">
      <property name="title">
       <string>Путь</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutPath">
       <item>
        <widget class="QLabel" name="labelStart">
         <property name="text">
          <string>Начало:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="startComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="labelEnd">
         <property name="text">
          <string>Конец:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="endComboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="findPathButton">
         <property name="text">
          <string>Найти путь</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearGraphButton">
         <property name="text">
          <string>Очистить граф</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBoxEdges">
      <property name="title">
       <string>Рёбра</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutEdges">
       <item>
        <widget class="QLabel" name="labelFrom">
         <property name="text">
          <string>От:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fromEdgeComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="labelTo">
         <property name="text">
          <string>К:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="toEdgeComboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="weightLineEdit">
         <property name="placeholderText">
          <string>Вес ребра</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addEdgeButton">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="resultTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
